doc/lsst.meas.algorithms/tasks/lsst.meas.algorithms.SubtractBackgroundTask.rst

```diff
@@ -0,0 +1,44 @@
+.. lsst-task-topic:: lsst.meas.algorithms.SubtractBackgroundTask
+
+######################
+SubtractBackgroundTask
+######################
+
+``SubtractBackgroundTask`` fits a model of the background of an exposure and subtracts it.
+
+.. _lsst.meas.algorithms.SubtractBackgroundTask-api:
+
+Python API summary
+==================
+
+.. lsst-task-api-summary:: lsst.meas.algorithms.SubtractBackgroundTask
+
+.. _lsst.meas.algorithms.SubtractBackgroundTask-subtasks:
+
+Retargetable subtasks
+=====================
+
+.. lsst-task-config-subtasks:: lsst.meas.algorithms.SubtractBackgroundTask
+
+.. _lsst.meas.algorithms.SubtractBackgroundTask-configs:
+
+Configuration fields
+====================
+
+.. lsst-task-config-fields:: lsst.meas.algorithms.SubtractBackgroundTask
+
+.. _lsst.meas.algorithms.SubtractBackgroundTask-debug:
+
+Debugging
+=========
+
+SubtractBackgroundTask has a debug dictionary containing three integer keys:
+
+unsubtracted
+  int; If >0: `fitBackground` displays the unsubtracted masked image overlaid with the grid of cells used to fit the background in the specified frame
```

Comment:
Please add periods at the end of these sentences, and put `int` in double-backticks.